crates/bevy_animation/src/_custom.rs

```diff
@@ -0,0 +1,2 @@
+// TODO: Custom implementations for types that are very common like Transforms!
+// TODO: Custom implementation for MorphTargets or something like it
```

Comment:
This sounds awesome but I would just remove the stumps for now.

Reply:
Is there so I don't forget to do it :P